### DISCUSSION TITLE
dose distribution of VMAT

### DISCUSSION THREAD

**Discussion by FasicolZyy** [e0404/matRad#634] [Q&A] [answered]
I would like to ask where I can see the MU jumps in matRad and how to obtain the changes in dose intensity on each curve of VMAT

**Comment by wahln** [comment]
Could you please elaborate? I don't really know what you are working with, because the official release of matRad does not support VMAT, there is only a research branch that is not fully functional... can't be of much help without more information.

**Reply by FasicolZyy** [reply]
I know that the delivery method of IMRT is usually step-and-shoot, delivering a prescribed dose at each control point with some fractions, while VMAT is a continuous dose delivery technique. I would like to know how the dose is quantitatively delivered along a continuous arc in VMAT, whether it is evenly delivered between two control points or varies in intensity. Can I obtain information about the dose variability between two control points in the matrad or dev-vmat branch?

**Reply by wahln** [reply, marked as answer]
I am not the best person to answer this, probably, as I was not deeply involved in the VMAT development and am no expert in this. I can give you some general answers:
In principle, VMAT continuously changes dose-rate and MLC shape. This is the delivery aspect. On the machine side, the standard approaches used the following approach, which is also implemented on the matRad branch:

1) We optimize an IMRT plan for a large number of gantry angles (that is, discrete points on the arc). We get a fluence modulation (expressed by a weight vector w in matRad).
2) The plan is segmented in a way that each discrete point get's an intensity and a single MLC shape. The sequencer is chosen to create closely connected shapes.
3) The shapes can be supersampled on the arc to create "more" points on the arc, somewhat similar to interpolation.
This discrete information is stored within matRad (after VMAT optimization in resultGUI.w and resultGUI.apertureInfo, I think). 
To my knowledge, a machine just interpolates linearly between these discrete points (control points). There's no link to a specific machine delivery in matRad, as it must not be used as a clinical treatment planning system.

The problem on the treatment planning side is now to also perform a continuous dose calculation / optimization to better represent the delivery.
@eric11210, who also developed the VMAT implementation on the branch, has described his approach in https://iopscience.iop.org/article/10.1088/1361-6560/aae65e which he also realized in matRad - it derives equations for calculating the continuous dose and also derivatives w.r.t. MLC movement. Currently, I don't remember / know if the continuous aspect is implemented as well on the matRad branch.

**Reply by FasicolZyy** [reply]
Thanks you very much,this helps me a lot
